### DISCUSSION TITLE
halo2 vs STARK？

### DISCUSSION THREAD

**Discussion by Demian101** [z2o-k7e/halo2-step-by-step#23] [Q&A]
@Kurt Pan: halo2 vs STARK老大难的原因主要是这俩背后都各有巨大的社区。路线之争了属于是，这个无解。

技术上看倒是融合的趋势，比如 https://github.com/maxgillett/halo2-fri-gadget ，plonky2/boojum，CCS 等。
以及可以探索这俩和folding/lookup两大主线的关系

@Zhouqi： 
1. 我们在探索 zkVM方向的risc0 vs zkWASM就是两个社区的路线一个大的路径了
2. 下次给大家汇报一下跑risc0的zeth结果，和zkWASM的大规模wasm的结果，我觉得有很多有意思的东西出来
3. 我可以贡献一期用和改zkWASM的一些心得 hhh

**Comment by Demian101** [comment]
如题

**Reply by voidcenter** [reply]
CCS是什么啊？  另外folding和lookup这两条线是互斥的还是正交的啊？
